channels/slack/src/main/kotlin/com/justai/jaicf/channel/slack/SlackChannel.kt

```diff
@@ -38,10 +40,21 @@ class SlackChannel private constructor(
     private constructor(botApi: BotApi, urlPrefix: String): this(botApi) {
         val config = SlackConfig().apply {
             methodsEndpointUrlPrefix = urlPrefix
+            methodsConfig = MethodsConfig().apply {
+                isStatsEnabled = false
+            }
         }
 
         val slack = Slack.getInstance(config)
-        app = App(AppConfig.builder().slack(slack).build())
+        app = App(AppConfig.builder()
+            .slack(slack)
+            .alwaysRequestUserTokenNeeded(false)
+            .singleTeamBotToken("empty")
```

Comment:
Why not to leave it empty?

Reply:
There are checks for empty strings in internal Slack filters. Leaving it empty or null will cause errors.